tox.ini

```diff
@@ -17,6 +17,7 @@ commands =
     # This repository uses a Markdown long_description, so the -r flag to
     # `setup.py check` is not needed. If your project contains a README.rst,
     # use `python setup.py check -m -r -s` instead.
+    python setup.py build_ext --antlr-runtime={env:ANTLR4_RUNTIME_TYPE:static}
```

Comment:
I think this should be changed to `pip install -e . --antlr-runtime={env:ANTLR4_RUNTIME_TYPE:static} --no-use-pep517`?

Reply:
Done.